planner/core/logical_plan_builder.go

```diff
@@ -1993,6 +2046,31 @@ func (b *PlanBuilder) extractAggFuncs(fields []*ast.SelectField) ([]*ast.Aggrega
 	return aggList, totalAggMapper
 }
 
+func (b *PlanBuilder) splitOuterAggFuncs(ctx context.Context, p LogicalPlan, aggFuncs []*ast.AggregateFuncExpr) (
```

Comment:
All the places that call it don't need the first return value. Could we remove it?